src/components/StreamBanner/StreamBanner.js

```diff
@@ -40,79 +42,85 @@ export default function StreamBanner ({
     avatarUrl = publicGlobe
     bannerUrl = allGroupsBanner
     // TODO list count of public posts and public groups in subtitle
-    subtitle = `All Posts Marked Public`
+    subtitle = 'All Posts Marked Public'
+  } else if (context === 'my') {
+    name = `My Home: ${capitalize(view)}`
+    avatarUrl = currentUser.avatarUrl || publicGlobe
```

Comment:
seems like we could have a better default than the globe, how about a house icon?

Reply:
To fulfill this, we'd need a specific home image in '/public'. There is likely an icon in the font set, but the rendering component is expecting an image, not an icon-class, so it would also need to be edited and given an alternative styling to conditionally use the Icon component.

Do you want to escalate to Aaron to produce a Home graphic? Note, this backup will only render when a user does not have an avatarUrl (do they get a default one?)

Reply:
users do get a default actually, so maybe this isnt an issue.